File: Tareas/Unidad2/a2253332351_practica02/11_SegundosPorDia.cpp
#include <iostream>
using namespace std;

int main() {
    double dias;
    double segundos;

    cout << "Ingrese los días: ";
    cin >> dias;

    segundos = dias * 24 * 60 * 60;

    cout << "Los días: " << dias << endl;
    cout << "Son equivalentes a: " << segundos << " segundos." << endl;

    return 0;
}

File: Tareas/Unidad2/a2253332351_practica02/11_SegundosPorDia.java
import java.util.Scanner;

public class SegundosPorDia {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double dias, segundos;

           System.out.println("Ingrese los días: ");
           dias = entrada.nextDouble();

           segundos = dias * 24 * 60 * 60;

           System.out.println("Los días: " + dias);
           System.out.println("Son equivalentes a: " + segundos + " segundos.");
     }
}
